st2common/st2common/bootstrap/runnersregistrar.py

```diff
@@ -521,6 +521,14 @@
             }
         },
         'runner_module': 'st2actions.runners.cloudslang.cloudslang_runner'
+    },
+    {
+        'name': 'noop',
+        'aliases': [],
+        'description': 'A runner that does absolutely nothing',
```

Comment:
Now that not true :)

It actually simply responds with a static response which does not change regardless of the input.